src/Common/tests/Cryptography/AlgorithmImplementations/RSA/RSAFactory.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Security.Cryptography.Rsa.Tests
+{
+    public interface IRSAProvider
+    {
+        RSA Create();
+        RSA Create(int keySize);
```

Comment:
Why is this interface necessary? I thought we had an RSA.Create that effectively did the same thing, using OpenSSL on Unix and CNG on Windows?